app.js

```diff
@@ -10,14 +10,25 @@ var http        = require('http'),
     archiver    = require('archiver'),
     queryString = require('querystring'),
     gitSync     = require('git-rev-sync'),
+    dotenv      = require('dotenv'),
     app         = express(),
     dirArray    = __dirname.split('/'),
     PORT        = 9001,
     PREFIX      = '',
     server,
     socket;
 
-require('dotenv').config();
+// Read in environment variables
+if (process.env.NODE_ENV === 'production') {
+  dotenv.config({path: '/etc/ood/config/apps/files/env'});
+} else {
+  dotenv.config({path: '.env.local'});
+}
```

Comment:
Instead how about we do:

```js
dotenv.config({path: '.env.local'});

if (process.env.NODE_ENV === 'production') {
  dotenv.config({path: '/etc/ood/config/apps/files/env'});
}
```

Thats closer to what happens on the dashboard right now and would cover small edge case where on development server we deploy two different shell or file explorer apps side by side and change the configuration of one using a .env.local for testing purposes. Thats the only two benefits though.

Reply:
Sure, although should be noted this change also needs to be made on https://github.com/OSC/ood-shell/pull/43